docs/support-pr-dossier.html

```
@@ -0,0 +1,383 @@
+        <div class="step"><div class="n">2</div><div class="b"><div class="cls">Mail::queue(SupportTicketConfirmationMail)</div><div class="d">Confirma o recebimento ao solicitante — <em>1× e independente do canal</em>. Fila <em>emails</em>.</div></div></div>
+        <div class="step"><div class="n">3</div><div class="b"><div class="cls">dispatch(DispatchSupportTicketJob)</div><div class="d">Joga o roteamento para a fila <em>default</em> (assíncrono).</div></div></div>
+      </div>
+    </div>
+
+    <div class="lane">
+      <div class="lane-h">Fase 2 — Roteamento · job, assíncrono</div>
+      <div class="steps">
+        <div class="step"><div class="n">4</div><div class="b"><div class="cls">TicketRouterService::dispatch()</div><div class="d">Orquestra: para cada canal de <em>category-&gt;destinationChannels()</em> cria um <em>TicketDestination</em> (estruturado para N destinos).</div></div></div>
+        <div class="step"><div class="n">5</div><div class="b"><div class="cls">app($type-&gt;senderClass())</div><div class="d">Resolve o handler por tipo. Canal novo = nova classe + 1 case no enum, <em>sem tocar no orquestrador</em>.</div></div></div>
+        <div class="step"><div class="n">6</div><div class="b"><div class="cls">EmailTicketSender::send() → DispatchResult</div><div class="d">Só envia (<em>sendNow</em>, inline no job) e devolve o resultado — não toca no banco. Monday segue stub (<em>pending</em>).</div></div></div>
+        <div class="step"><div class="n">7</div><div class="b"><div class="cls">$destination->update($result)</div><div class="d">O orquestrador grava o desfecho no destino. O <em>chamado segue Pendente</em> — notificar o setor não avança o ciclo de vida.</div></div></div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)